pull-request.sh

```diff
@@ -90,6 +91,20 @@ main () {
     fi
     echo "Pull requests will go to ${PULL_REQUEST_BRANCH}"
 
+    if [ -z "${PULL_REQUEST_DRAFT}" ]; then
+        echo "No explicit preference for draft PR; defaulting to normal PR."
+        PULL_REQUEST_DRAFT="false"
+    else
+        PULL_REQUEST_DRAFT="true"
+    fi
+
+    # separate block for logging draft vs not draft
+    if [ "$PULL_REQUEST_DRAFT" = "true" ]; then
+        echo "Created PRs will be draft PRs."
+    else 
+        echo "Created PRs will be normal PRs."
+    fi
+
```

Comment:
This second block (above the comment) is actually a bit redundant, any reason to not just print these in the first check, where you set true/false?